src/config/print.config.js

```diff
@@ -0,0 +1,5 @@
+// 90 Was the old mapviewer magic number set to compensate for the DPI difference between the
+// mapviewer and the service print. Currently, 144 seems to be giving better results at keeping
+// the same scale overall.
+// We're going to check thoroughly if this works as intended, or if this is a monitor dpi setting element.
+export const PRINT_MAGIC_NUMBER = 144
```

Comment:
I would maybe not call it "magic number" but something that self-explains what it does.
Maybe `PRINT_DPI_COMPENSATION`?